eclair-core/pom.xml

```diff
@@ -270,6 +270,11 @@
             <version>1.5.0</version>
         </dependency>
         <!-- MONITORING -->
+        <dependency>
+            <groupId>io.kamon</groupId>
+            <artifactId>kamon-prometheus_${scala.version.short}</artifactId>
+            <version>2.5.4</version>
```

Comment:
This should have used `${kamon.version}`, and also the dependency should have been put in `eclair-node/pom.xml`, (right below `kamon-apm-reporter`), not `eclair-core/pom.xml`

Reply:
Good catch! @GoutamVerma can you fix that in a new PR?

Reply:
@pm47 thank you for pointing out, I will fix that in new PR.